src/eos-updater-ctl

```diff
@@ -293,6 +297,14 @@ def command_server(server_command, quiet=False):
     return 0
 
 
+def nonnegative_int(value):
+    int_value = int(value)
+    if int_value < 0:
+        raise argparse.ArgumentTypeError(
+            '%s must be a non-negative integer' % value)
```

Comment:
The error message here will read:

```
eos-updater-ctl: error: argument --timeout: -1 must be a non-negative integer
```

which I don't really think makes sense. I think `'%s is negative'` would be clearer. In the case where it's not an integer the error will be:

```
eos-updater-ctl: error: argument --timeout: invalid nonnegative_integer value: 'snake'
```

You could just `raise ValueError` in the `< 0` case and the default message would be fine:

```
eos-updater-ctl: error: argument --timeout: invalid nonnegative_integer value: '-1'
```